src/engine/controls/loopingcontrol.cpp

```diff
@@ -958,6 +959,14 @@ void LoopingControl::updateBeatLoopingControls() {
 }
 
 void LoopingControl::slotBeatLoop(double beats, bool keepStartPoint, bool enable) {
+
+    // if a seek was queued in the engine buffer move the current sample to its position
+    ControlValueAtomic<double> seekPosition;
```

Comment:
we can use here a plain double, because it lives on the stack and is not used concurrently.